ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json

```diff
@@ -5540,121 +4373,164 @@
         "hide": 0,
         "includeAll": true,
-        "label": "Queue",
+        "label": "vhost",
         "multi": true,
-        "name": "queue",
+        "name": "vhost",
         "options": [],
-        "query": "label_values(rabbitmq_queue_messages_published_total{namespace=~\"$namespace\", pod=~\"$pod\"}, queue)",
-        "refresh": 2,
+        "query": {
+          "query": "label_values(rabbitmq_queue_messages_published_total{namespace=~\"$namespace\",pod=~\"$pod\"}, queue_vhost)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
         "regex": "",
         "skipUrlSync": false,
         "sort": 0,
-        "tagValuesQuery": "",
-        "tags": [],
-        "tagsQuery": "",
-        "type": "query",
-        "useTags": false
+        "type": "query"
       },
       {
-        "allValue": null,
         "current": {
-          "selected": true,
+          "selected": false,
           "text": "All",
-          "value": ["$__all"]
+          "value": "$__all"
         },
-        "datasource": "$ds_prometheus",
-        "definition": "label_values(rabbitmq_queue_messages_published_total{namespace=~\"$namespace\", pod=~\"$pod\"}, queue_vhost)",
+        "datasource": {
+          "type": "prometheus",
+          "uid": "P0D6E4079E36703EB"
+        },
+        "definition": "label_values(rabbitmq_queue_messages_published_total{namespace=~\"$namespace\",pod=~\"$pod\",queue_vhost=~\"$vhost\"}, queue)",
         "hide": 0,
         "includeAll": true,
-        "label": "VHost",
-        "multi": true,
-        "name": "queue_vhost",
+        "label": "Queue",
+        "multi": false,
+        "name": "queue",
         "options": [],
-        "query": "label_values(rabbitmq_queue_messages_published_total{namespace=~\"$namespace\", pod=~\"$pod\"}, queue_vhost)",
-        "refresh": 2,
+        "query": {
+          "query": "label_values(rabbitmq_queue_messages_published_total{namespace=~\"$namespace\",pod=~\"$pod\",queue_vhost=~\"$vhost\"}, queue)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
         "regex": "",
         "skipUrlSync": false,
         "sort": 0,
-        "tagValuesQuery": "",
-        "tags": [],
-        "tagsQuery": "",
-        "type": "query",
-        "useTags": false
+        "type": "query"
+      },
+      {
+        "datasource": {
+          "type": "prometheus",
+          "uid": "P0D6E4079E36703EB"
+        },
+        "filters": [],
+        "hide": 0,
+        "name": "Filters",
+        "skipUrlSync": false,
+        "type": "adhoc"
       }
     ]
   },
   "time": {
-    "from": "now-3h",
+    "from": "now-1h",
     "to": "now"
   },
   "timepicker": {
-    "refresh_intervals": ["15s", "30s", "1m", "5m", "10m"],
-    "time_options": ["5m", "15m", "1h", "6h", "12h", "24h", "2d", "7d", "30d"]
+    "refresh_intervals": [
+      "15s",
```

Comment:
We have a 30-second scrape interval as usual, so a fast refresh doesn't make sense.

Reply:
Changed the refresh intervals as in the other dashboards.